.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "src/software/simulation/ssl_simulation_protocol"]
-	path = src/software/simulation/ssl_simulation_protocol
-	url = git@github.com:UBC-Thunderbots/SSL-Simulation-Protocol-Support.git
```

Comment:
Was this only for virtual robocup?

Reply:
I can't explain to you why this exists. It seems to be used in our old simulator as a linking library between our firmware and our simulator (but then we changed simulators and firmware)

I went through our code history and I can't find a single commit that uses this git submodule

Reply:
I removed it because it doesn't compile properly and prevents us from running `bazel test //software/...`